README.md

```diff
@@ -54,14 +54,24 @@ Bei der Ausführung eines Unterprojekts via `run` können per `-Dargs="Argument1
 
     ./gradlew :test-client:run -Dargs="--player1 ../../player/build/libs/defaultplayer.jar --player2 ../../player/build/libs/defaultplayer.jar --tests 3"
 
+### Tests
+
+Unsere Unittests nutzen das [Kotest-Framework](https://kotest.io) mit [JUnit](https://junit.org) im Hintergrund.
+
+Dabei setzen wir auf die [ShouldSpec](https://kotest.io/styles/#should-spec) und ggf. [BehaviorSpec](https://kotest.io/styles/#behavior-spec).
+Bisherige Tests nutzen die StringSpec, welche jedoch wegen fehlendem nesting auf Dauer zu unübersichtlich ist.
+
 ### Arbeiten mit IntelliJ IDEA
 
 Zuerst sollte sichergestellt werden, dass die neuste Version von IntelliJ IDEA verwendet wird, da es ansonsten Probleme mit Kotlin geben kann.
 
 In IntelliJ kann man das Projekt bequem von Gradle importieren, wodurch alle Module und Bibliotheken automatisch geladen werden.
 Dazu sind folgende Schritte notwendig:
 
-- Projekt klonen: `git clone --recurse-submodules git@github.com:CAU-Kiel-Tech-Inf/backend.git`
+- Projekt klonen:
+  ```sh
+  git clone git@github.com:CAU-Kiel-Tech-Inf/backend.git --recurse-submodules --shallow-submodules
```

Comment:
I think shallow-submodules makes it impossible to change the commit of the backend submodule for testing. There should be a note here which says how to change that if one wants to change the backend commit.

Reply:
No, it doesn't. If you fetch it will still fetch the branches - I have already done exactly that. It simply avoids unnecessary fetching for people that never mess with it.

Reply:
Instructions are the same in the gui btw, that's why I changed it.

Reply:
Okay, you're right.